pkg/registry/configmap/etcd/etcd.go

```diff
@@ -36,7 +37,13 @@ func NewREST(opts generic.RESTOptions) *REST {
 
 	newListFunc := func() runtime.Object { return &api.ConfigMapList{} }
 	storageInterface := opts.Decorator(
-		opts.Storage, 100, &api.ConfigMap{}, prefix, configmap.Strategy, newListFunc, storage.NoTriggerPublisher)
+		opts.Storage,
+		cachesize.GetWatchCacheSizeByResource(cachesize.ConfigMaps),
```

Comment:
I just unified it for the remaining 3 resources in this PR (we already have this pattern for other 20 or so).
I'm happy to change that, but I would prefer to do it in a separate PR.